### PR TITLE
Fix #5, #6

### LINKED ISSUE #5: Wrong port for IIS Express in launchSettings.json
By default, VS runs the project using IIS Express, which is configured to use 44359, while the Kestrel configuration, as well as the documentation and pre-configured CallbackUri in appsettings.json use 5001. As a result, authentication doesn't work.

### PR DESCRIPTION
**Pull request by Leon99**
Fix #5, #6

### CODE REVIEW

File: XeroNetStandardApp/Controllers/EmployeesInfoController.cs [hunk 1]
```diff
@@ -42,7 +42,7 @@ public async Task<ActionResult> Index()
       string accessToken = xeroToken.AccessToken;
       string xeroTenantId = xeroToken.Tenants[0].TenantId.ToString();
 
-      var PayrollAUApi = new PayrollAUApi();
+      var PayrollAUApi = new PayrollAuApi();
```
Comment (igitur) [inline XeroNetStandardApp/Controllers/EmployeesInfoController.cs:45]: Duplicate of #4

**Comment by igitur** [comment]
Some of these have already been addressed in #3  and #4 .

**Review by jenksguo — APPROVED** [review]

**Comment by jenksguo** [comment]
Hi @igitur, thanks for doing the PR. This was neglected for a while by me and your PR fixed it. 

I would love to give you some Xero developer swags could you send your delivery address to my email? jenks.guo@xero.com

**Comment by igitur** [comment]
@jenksguo

Yay, awesome. I'll send you an email. Thanks so much.

tbh, this PR was simple. I think my contribution at https://github.com/XeroAPI/xoauth/pull/14 was more worthy of merch. ;-)

**Comment by jenksguo** [comment]
Hi @Leon99 thanks for your contribution! Could you also send your address to me via email? jenks.guo@xero.com

**Pull request merged**